internal/jsre/deps/web3.js

```diff
@@ -5490,6 +5490,10 @@ var properties = function () {
         new Property({
             name: 'protocolVersion',
             getter: 'eth_protocolVersion'
+        }),
+        new Property({
+          name: 'config',
+          getter: 'eth_config'
```

Comment:
Shouldn't this be defined in `internal/web3ext/web3ext.go`? We want to modify the web3js as little as possible iirc

Reply:
good catch - moved over now